### PR TITLE
Contact main html jordan

### PR DESCRIPTION
**Pull request by jordanleeprogram**

### CODE REVIEW

File: src/contact.html [hunk 1]
```diff
@@ -15,16 +15,16 @@
 
       <form>
         <label for="fname">First Name</label>
-        <input type="text" id="fname" name="firstname" placeholder="Enter first name...">
+        <input type="text"  name="firstname" placeholder="Enter first name...">
```
Comment (mz106) [inline src/contact.html:18]: Conflict from removing id

File: src/contact.html [hunk 2]
```diff
@@ -15,16 +15,16 @@
...
-        <input type="text" id="fname" name="firstname" placeholder="Enter last name...">
+        <input type="text"  name="lastname" placeholder="Enter last name...">
         
         <label for="fname">Your E-mail</label>
-        <input type="text" id="fname" name="firstname" placeholder="Enter your e-mail...">
+        <input type="text"  name="email" placeholder="Enter your e-mail...">
 
         <label for="fname">Your Mobile Number</label>
-        <input type="text" id="fname" name="firstname" placeholder="Enter your phone number...">
+        <input type="text"  name="phone" placeholder="Enter your phone number...">
```
Comment (mz106) [inline src/contact.html:27]: Conflict from removing id

**Review by mz106 — CHANGES_REQUESTED** [review]
Conflicts - please resolve

File: src/contact.html [hunk 3]
```diff
@@ -15,16 +15,16 @@
 
       <form>
         <label for="fname">First Name</label>
-        <input type="text" id="fname" name="firstname" placeholder="Enter first name...">
+        <input type="text"  name="firstname" placeholder="Enter first name...">
        
         <label for="fname">Last Name</label>
-        <input type="text" id="fname" name="firstname" placeholder="Enter last name...">
+        <input type="text"  name="lastname" placeholder="Enter last name...">
```
[the remark this answers is not in the document]
Comment (jordanleeprogram) [reply src/contact.html:21]: should i replace all IDs then and re-push?

[the remark this answers is not in the document]
Comment (jordanleeprogram) [reply src/contact.html:31]: all good

File: src/contact.html [hunk 4]
```diff
@@ -15,16 +15,16 @@
...
         
         <label for="fname">Your E-mail</label>
-        <input type="text" id="fname" name="firstname" placeholder="Enter your e-mail...">
+        <input type="text"  name="email" placeholder="Enter your e-mail...">
```
[the remark this answers is not in the document]
Comment (jordanleeprogram) [reply src/contact.html:34]: good

File: src/contact.html (hunk 2 above)
Comment (jordanleeprogram) [reply src/contact.html:37]: good

**Review by mz106 — APPROVED** [review]
Approved

**Pull request merged**